pkg/operator/csi/csicontrollerset/csi_controller_set.go

```diff
@@ -271,8 +270,10 @@ func (c *CSIControllerSet) WithStorageClassController(
 
 // New returns a basic *ControllerSet without any controller.
 func NewCSIControllerSet(operatorClient v1helpers.OperatorClientWithFinalizers, eventRecorder events.Recorder) *CSIControllerSet {
-	return &CSIControllerSet{
+	c := &CSIControllerSet{
```

Comment:
nit:
This could remain as anonymous structure pointer return
```go
return &CSIControllerSet{
  operatorClient: operatorClient,
  eventRecorder: eventRecorder,
  conditionalStaticResourcesControllers: []factory.Controller{},
}
```

Reply:
Done